### PR TITLE
Java17 sp3

### PR DESCRIPTION
**Pull request by ddelblanco**
Added Java 17 and SpringBoot 3.2.2
Removing the use of cookies. 
Changed the AGS services to use a token (just for our demo purposes, not the way to go in prod, for prod we need to use the onetime token and then trade it and refresh it later as it happens in the not demo path)

### CODE REVIEW

**Comment by mgwozdz-unicon** [comment]
What is the purpose of the new AGS Advantage Token?

File: src/main/java/net/unicon/lti/utils/lti/LTI3Request.java [hunk 1]
```diff
@@ -512,17 +478,7 @@ public String processRequestParameters(HttpServletRequest request, Jws<Claims> j
...
-            return "true";
-        } else {
-            if (complete) {
-                isComplete = "";
-            } else if (correct) {
-                isCorrect = "";
-            }
-            return isComplete + isCorrect;
-        }
+        return (complete && correct);
```
Comment (mgwozdz-unicon) [inline src/main/java/net/unicon/lti/utils/lti/LTI3Request.java:481]: ❤️

File: src/main/resources/static/css/bootstrap.min.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*!
- * Bootstrap v4.2.1 (https://getbootstrap.com/)
+ * Bootstrap v4.1.3 (https://getbootstrap.com/)
```
Comment (mgwozdz-unicon) [inline src/main/resources/static/css/bootstrap.min.css:2]: Was this file down graded?

Comment (ddelblanco) [reply src/main/resources/static/css/bootstrap.min.css:2]: Interesting... I thought I downloaded the last version....  I'll check that

**Comment by ddelblanco** [comment]
About the PR AGS Advantage Token... I've removed the use of session cookies for our AGS UX, so the token is just a way to secure and pass the info we need to the AGS endpoints. As we don't have a session mechanism for our tool now, this token is supposed to make it "not session dependent" but still keep the endpoints kind of "secured".  This is not what we would use in a prod tool, but it is enough for the demo purposes.

**Pull request merged**